install/installer/pkg/common/render_test.go

```diff
@@ -28,6 +29,46 @@ func TestCompositeRenderFunc_NilObjectsNilError(t *testing.T) {
 	require.Len(t, objects, 0)
 }
 
+func TestReplicas(t *testing.T) {
+	testCases := []struct {
+		Component        string
+		ExpectedReplicas int32
+	}{
+		{
+			Component:        "server",
+			ExpectedReplicas: 123,
+		},
+		{
+			Component:        "dashboard",
```

Comment:
```suggestion
			Component:        dashboard.Component,
```
and similar in other component entries. The test can be used as an example how to specify it so should also "do it as we'd like it to be done"

Reply:
[29d51e5](https://github.com/gitpod-io/gitpod/pull/9495/commits/29d51e56a404ab9f96eb55f0b00e888bebf02509)